src/test/scala/ch/epfl/bluebrain/nexus/tests/admin/AdminSpec.scala

```diff
@@ -576,5 +592,154 @@ class AdminSpec extends BaseSpec with OptionValues with CancelAfterFailure with
+              "_source" -> Json.obj(
+                "@context"    -> Json.fromString(config.prefixes.coreContext.toString),
+                "@id"         -> Json.fromString(s"$adminBase/projects/$id"),
+                "@type"       -> Json.fromString("nxv:Project"),
+                "_deprecated" -> Json.fromBoolean(false),
+                "_rev"        -> Json.fromInt(1),
+                "config" -> Json.obj(
+                  "maxAttachmentSize" -> Json.fromInt(10)
+                ),
+                "description" -> Json.fromString(s"$id description"),
+                "name"        -> Json.fromString(id),
+                "prefixMappings" -> Json.arr(
+                  Json.obj(
+                    "namespace" -> Json.fromString("https://bbp-nexus.epfl.ch/vocabs/nexus/core/terms/v0.1.0/"),
+                    "prefix"    -> Json.fromString(s"nxv-$id")
+                  ),
+                  Json.obj(
+                    "namespace" -> Json.fromString("https://shapes-registry.org/commons/person"),
+                    "prefix"    -> Json.fromString(s"person-$id")
+                  )
+                )
+              )
+            )
+          }: _*
+        )
+      }
+
+      "create project and grant project permissions to user" in {
+        val permissionsJson = jsonContentOf("/iam/add.json", replSub + (quote("{perms}") -> "projects/read")).toEntity
+
+        forAll(projectIds) { id =>
+          cl(
+            Req(PUT,
+                s"$adminBase/projects/$id",
+                headersUser,
+                reqEntity(nxv = s"nxv-$id", person = s"person-$id", description = s"$id description", name = id)))
+            .mapJson { (json, result) =>
+              result.status shouldEqual StatusCodes.Created
+              json shouldEqual createRespJson(id, 1L)
+            }
+
+          cl(Req(PUT, s"$iamBase/acls/$id", headersGroup, permissionsJson)).mapResp { result =>
+            result.status shouldEqual StatusCodes.OK
+            result.entity.isKnownEmpty() shouldEqual true
+          }
+
+          //Get permission added
+          eventually {
+            cl(Req(uri = s"$adminBase/projects/$id/acls", headers = headersUser)).mapJson { (json, result) =>
+              result.status shouldEqual StatusCodes.OK
+              json shouldEqual jsonContentOf(
+                "/iam/project-perms-response.json",
+                resourceIamCtx ++ replSub + (quote("{perms}") -> "projects/read", quote("{path}") -> id))
+            }
+          }
+        }
+      }
+
+      "return only projects the user has access to" in {
+        val expectedResults = Json.obj(
+          "@context" -> Json.fromString(config.prefixes.searchContext.toString),
+          "_results" -> projectListingResults(projectIds),
+          "_total"   -> Json.fromInt(projectIds.size),
+          "_links" -> Json.obj(
+            "_self" -> Json.fromString(s"$adminBase/projects")
```

Comment:
Why is this self still here? I thought we removed this predicate

Reply:
I think we agreed to remove it from source and single resource view, but still leave it in search results?

Reply:
From my point of view it adds no value anywhere